src/Microsoft.AspNetCore.Mvc.Core/Internal/DefaultModelBindingContext.cs

```diff
@@ -130,32 +132,16 @@ public DefaultModelBindingContext()
         }
 
         /// <inheritdoc />
-        public override NestedScope EnterNestedScope()
```

Comment:
I moved about 10 lines of code to comply with our coding standards. If I hadn't moved it, there's a good chance you might have suggested I move it :see_no_evil: